Dockerfile

```diff
@@ -23,3 +23,5 @@ RUN bundle install
 
 COPY package.json yarn.lock ./
 RUN yarn install
+
+RUN bundle exec rake webpacker:compile
```

Comment:
Webpacker compile is not necessary in development mode, as it does this when you request the assets.

Reply:
Moved to readme for troubleshooting if needed.  I ran into the same issue and had to manually compile into my web docker container.